### PR TITLE
Setup translation infrastructure

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

**Comment by stejskalleos** [comment]
Thanks @ofedoren

File: locale/cs_CZ/hammer-cli-foreman-puppet.po.time_stamp
Comment (ekohl) [inline locale/cs_CZ/hammer-cli-foreman-puppet.po.time_stamp:1]: Should the time stamp files actually be stored in git? I don't think we do that anywhere else.

File: locale/cs_CZ/hammer-cli-foreman-puppet.edit.po
Comment (ekohl) [inline locale/cs_CZ/hammer-cli-foreman-puppet.edit.po:1]: The edit files are intermediate files and don't belong in git. They should be in `.gitignore`.

**Pull request merged**